app/views/coronavirus_pages/shared/_github.html.erb

```diff
@@ -1,18 +1,18 @@
 <div class="govuk-!-padding-bottom-1">
   <%= render "govuk_publishing_components/components/heading", {
-  text: "1. Update GitHub",
+  text: t("coronavirus_pages.github.update_heading"),
   padding: true
   } %>
   <p class="govuk-body">
-     Update the<%= link_to " coronavirus content on GitHub", github_url %>
+     <%= "#{t("coronavirus_pages.github.update_sub_heading")}#{ link_to " coronavirus content on GitHub", github_url }" %>
```

Comment:
Sorry I missed this on my last review. It looks a bit suspect that we've got half of this header in the locale file and half here in the link. It caught my eye when I saw ["Update the"](https://github.com/alphagov/collections-publisher/pull/1217/files#diff-44438ce218f5287c58d0017f965d888715635d94280669896f75841fbd7b4cd7R18) in the locale file and thought that was a mistake.

As this contains HTML this seems like it might be more complex than every other translation 🐼 . I'd suggest perhaps changing the key to have a [`_html` suffix](https://guides.rubyonrails.org/i18n.html#using-safe-html-translations) and setting it as:
 ```
  update_sub_heading_html: "Update the <a href=\"%{github_url}\">coronavirus content on Github</a>"
```

So that we can do 

```
<%= t("coronavirus_pages.github.update_sub_heading_html", github_url: github_url) %>
```

in the view.

Reply:
Nice! Thanks @kevindew have updated.